src/components/ListTodo.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-6">
+        <div class="todolist not-done">
+          <h1>Todos</h1>
+          <input type="text" class="form-control add-task" placeholder="Add task">
+          <hr>
+          <ul id="sortable" class="list-unstyled">
+            <li class="row ui-state-default" v-for="(task, index) in tasksNotDone" v-bind:key="index">
+              <div class="col-md-9">
+                {{ task.content }}
+              </div>
+              <div class="col-md-3">
+                <button class="btn btn-default">
+                  <i class="fa fa-edit"></i>
+                </button>
+                <button class="btn btn-default">
+                  <i class="fa fa-check"></i>
+                </button>
+              </div>
+            </li>
+          </ul>
+          <div class="todo-footer">
+            <strong><span class="count-todos">{{ tasksNotDone.length }}</span></strong> Items Left
+          </div>
+        </div>
+      </div>
+      <div class="col-md-6">
+        <div class="todolist">
+          <h1>Already Done</h1>
+          <ul id="done-items">
+            <li class="row" v-for="(task, index) in tasksIsDone" v-bind:key="index">
+              <div class="col-md-10">
+                {{ task.content }}
+              </div>
+              <div class="col-md-2">
+                <button class="btn btn-default"><i class="fa fa-times"></i></button>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import TodoService from '@/services/todo'
+export default {
+  name: 'ListTodo',
+  data () {
+    return {
+      tasks: []
+    }
+  },
+  created () {
+    TodoService.getTodoList().then((response) => {
+      if (response.status === 200) {
+        this.tasks = response.body.data
+      }
+    })
+  },
+  computed: {
+    tasksNotDone () {
+      return this.tasks.filter((item) => { return !item.is_done })
```

Comment:
Ở đây a viết như thế này có được không nhé
```js
return this.tasks.filter(task => task.is_done)
```
Tìm hiểu sự khác nhau xem sao nhé.

Reply:
Dạ với cái này thì dùng cho function có duy nhất 1 câu lệnh, còn nhiều hơn thì dùng { ... }

Reply:
Còn 1 điểm nữa. 2 cách viết như bên dưới cho kết quả không giống nhau đâu e.
```js
this.tasks.filter(task => !task.is_done)
this.tasks.filter(task => {!task.is_done})
```

Reply:
`this.tasks.filter(task => {!task.is_done})` thì nó trả về rỗng vì ko return các task ạ

Reply:
@LeCaoDat E biết vì sao ko?

Reply:
@trandaison Dạ vì cái {} cần return ạ

Reply:
@trandaison dạ e nghĩ vì nó là cái callback nên cần phải return giá trị sau khi chạy xong thì mới lấy được kết quả.

Reply:
@trandaison dạ e nghĩ vì nó là cái callback nên cần phải return giá trị thì mới lấy được kết quả.

Reply:
```js
// 2 cách viết tương tự
this.tasks.filter(task => !task.is_done)
this.tasks.filter(task => {return !task.is_done})

// khác với cách viết dưới
this.tasks.filter(task => {!task.is_done})
```
Trong arrow function, nếu sử dụng cặp ngoăc `{ }` thì phải dùng từ khoá `return` nếu cần hàm return về 1 giá trị.
Nếu hàm đơn giản có thể viết chung 1 hàng (như cách viết đầu tiên), thì hàm sẽ mặc định return về giá trị của biến/hàm nằm ở thân hàm (`!task.is_done`).

Còn dĩ nhiên callback truyền vào hàm `filter` phải return về một giá trị thì mới "lọc" được chứ.

Reply:
Dạ em hiểu rồi ạ